tests/test_generators/test_pydanticgen.py

```diff
@@ -1487,15 +1495,18 @@ def expectation(self):
     [TestCase(type="pass", array=np.zeros((3, 4, 5, 6), dtype=dt)) for dt in (int, float, str)]
     + [TestCase(type="fail", array=a) for a in (4, 3.0, "three")],
 )
-@pytest.mark.parametrize("representation", [[ArrayRepresentation.LIST], [ArrayRepresentation.NPARRAY]])
+@pytest.mark.parametrize(
+    "representation",
+    [[ArrayRepresentation.LIST], pytest.param([ArrayRepresentation.NUMPYDANTIC], marks=pytest.mark.pydantic_npd)],
```

Comment:
this is very not DRY, i'm going to split out the array representation parameterization and reuse it or maybe make it into a parameterized fixture instead of copy pasting this a million times

Reply:
done in https://github.com/linkml/linkml/pull/2178/commits/7ff138e7d9ab25bc64f51312210c1a41f93162d0